assets/css/base/_index.scss

```
@@ -0,0 +1,8 @@
+body {
+  padding-top: 40px;
```

Small comment, but we tend to use the `rem` function from our `_sizing.scss`. 

```suggestion
  padding-top: rem(40px);
```